examples/gallery/embellishments/solar.py

```diff
@@ -15,39 +15,24 @@
 import pygmt
 
 fig = pygmt.Figure()
-# Create a figure showing the global region on a Mollweide projection
-# Land color is set to dark green and water color is set to light blue
-fig.coast(region="d", projection="W0/15c", land="darkgreen", water="lightblue")
-# Set a time for the day-night terminator and twilights, 17:00 UTC on
-# January 1, 2000
+# Create a global map with central longitude 0° East ("d") using a Mollweide ("W")
+# projection with a width of 15 centimeters
+fig.basemap(region="d", projection="W15c", frame=True)
+fig.coast(land="darkgreen", water="lightblue", shorelines="1/1p,gray50")
```

Comment:
No need to add shorelines:
```suggestion
fig.coast(land="darkgreen", water="lightblue")
```

Reply:
Removed in commit https://github.com/GenericMappingTools/pygmt/pull/3858/commits/b04abfd8ce5feca6143d3bff8d25a7f0a1974747.
(accidentally added while play with the script 🙁).